hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -316,6 +326,16 @@
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Slow\] \
                                          --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
                 export GINKGO_PARALLEL="y"
+        - 'gke-ingress-release-1.2':
+            description: 'Run [Feature:Ingress] tests on GKE on the release-1.2 branch.'
+            timeout: 90
+            emails: 'beeps@google.com'
+            test-owner: 'beeps'
+            job-env: |
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
+                export PROJECT="kubernetes-gke-ingress"
```

Comment:
Also this project has the gke side flag we need to flip with an internal cl each time a new e2e gke project is created